src/component/TodoItem.jsx

```diff
@@ -61,7 +61,7 @@ const TodoItem = ({ todo, updateTodo, deleteTodo }) => {
 
 TodoItem.propTypes = {
   todo: PropTypes.shape({
-    id: PropTypes.number.isRequired,
+    id: PropTypes.string.isRequired,
```

Comment:
fix a nasty warning in the console
![Screenshot 2024-09-21 at 12 46 47](https://github.com/user-attachments/assets/8649ce64-0167-4c65-9cea-ac4b430ad0b2)